plugins/shortcuts/src/Shortcuts.tsx

```diff
@@ -26,7 +26,7 @@ import { shortcutsApiRef } from './api';
 const useStyles = makeStyles({
   root: {
     flex: '1 1 auto',
-    overflow: 'scroll',
+    'overflow-y': 'scroll',
```

Comment:
the canonical form in react land here is `overflowY: 'scroll'`, and that's what typescript suggests

Reply:
ah thanks, updated